string cam_id
uint64 framenumber
float64 timestamp
MTCalPoint[] points
